app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
+  before_action :configure_sign_up_params, only: [:create]
+  before_action :configure_account_update_params, only: :update
+
+  # GET /resource/sign_up
+  def new
+    super
+  end
+
+  #POST /resource
+  def create
+    super do |resource|
+      if resource.valid?
+        @organization = Organization.create(name: params[:organization_name])      
+        resource.update(organization_id: @organization.id)
+    end
+  end
+
+  # GET /resource/edit
+  def edit
+    super
+  end
+
+  # PUT /resource
+  def update
+    super
+  end
+
+  # DELETE /resource
+  def destroy
+    super
+  end
+
+  # GET /resource/cancel
+  # Forces the session data which is usually expired after sign
+  # in to be expired now. This is useful if the user wants to
+  # cancel oauth signing in/up in the middle of the process,
+  # removing all OAuth session data.
+  def cancel
```

Comment:
no need to create these methods really, just delete or comment all unmodified methods